plugin-libs/plugin-command/src/main/java/com/salesforce/bazel/eclipse/command/BazelWorkspaceCommandRunner.java

```diff
@@ -264,7 +264,7 @@ public File computeBazelWorkspaceOutputBase() {
         if (bazelOutputBaseDirectory == null) {
             try {
                 ImmutableList.Builder<String> argBuilder = ImmutableList.builder();
-                argBuilder.add("info").add("execution_root");
+                argBuilder.add("info").add("output_base");
```

Comment:
This fixed the particular "jar not found" issue I was seeing.  I have used this change all day without issues, and it also matches the method name ... (?)

Reply:
And it was changed recently in PR #75 so matches the timeline of regression.